lib/Resizable.js

```diff
@@ -182,6 +186,27 @@ export default class Resizable extends React.Component<Props, State> {
       const canDragX = (this.props.axis === 'both' || this.props.axis === 'x') && ['n', 's'].indexOf(axis) === -1;
       const canDragY = (this.props.axis === 'both' || this.props.axis === 'y') && ['e', 'w'].indexOf(axis) === -1;
 
+      // Check if handle's position has moved since last resizeHandler call - adjust deltas accordingly
+      if (this.draggingNode == null && e.target instanceof HTMLElement) {
+        this.draggingNode = e.target;
+      }
+      if (this.draggingNode instanceof HTMLElement) {
+        const handleRect = this.draggingNode.getBoundingClientRect();
+        if (this.lastTop != null && this.lastLeft != null) {
+          const deltaLeftSinceLast = handleRect.left - this.lastLeft;
+          const deltaTopSinceLast = handleRect.top - this.lastTop;
+          
+          if (canDragX && axis.indexOf('w') > -1) {
+            deltaX += deltaLeftSinceLast / this.props.transformScale;
+          }
+          if(canDragY && axis.indexOf('n') > -1) {
```

Comment:
See above comment, `axis[0] === 'n'` will work here.

Reply:
Very true, much cleaner this way 👌